Cargo.toml

```diff
@@ -26,10 +26,11 @@ futures = "0.3.30"
 futures-util = "0.3.30"
 hyper = { version = "1.3.1", features = ["full"] }
 hyper-util = "0.1.3"
+# NOTE (@Techassi): We cannot bump to 2.0.0 because kube uses 1.X.X
 json-patch = "1.4.0"
-k8s-openapi = { version = "0.21.1", default-features = false, features = ["schemars", "v1_29"] }
+k8s-openapi = { version = "0.22.0", default-features = false, features = ["schemars", "v1_29"] }
```

Comment:
Looking at the supported product versions
![image](https://github.com/stackabletech/operator-rs/assets/29303194/69c3d926-4c48-46ce-ba66-8cd5dc5f64de)
I think there is a high chance we want to support 1.30 in the next release, so I would be in favor of bumping to 1_30

Reply:
Sure, if other people are fine with it. I don't know if this introduces breaking changes tho.

Reply:
These are the required changes:

```diff
diff --git a/Cargo.toml b/Cargo.toml
index d902bd3..7e779e6 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -28,7 +28,7 @@ hyper = { version = "1.3.1", features = ["full"] }
 hyper-util = "0.1.3"
 # NOTE (@Techassi): We cannot bump to 2.0.0 because kube uses 1.X.X
 json-patch = "1.4.0"
-k8s-openapi = { version = "0.22.0", default-features = false, features = ["schemars", "v1_29"] }
+k8s-openapi = { version = "0.22.0", default-features = false, features = ["schemars", "v1_30"] }
 # We use rustls instead of openssl for easier portablitly, e.g. so that we can build stackablectl without the need to vendor (build from source) openssl
 kube = { version = "0.91.0", default-features = false, features = ["client", "jsonpatch", "runtime", "derive", "rustls-tls"] }
 lazy_static = "1.4.0"
diff --git a/crates/stackable-operator/src/builder/pod/security.rs b/crates/stackable-operator/src/builder/pod/security.rs
index 24896da..9012653 100644
--- a/crates/stackable-operator/src/builder/pod/security.rs
+++ b/crates/stackable-operator/src/builder/pod/security.rs
@@ -394,7 +394,8 @@ mod tests {
                     gmsa_credential_spec_name: Some("name".to_string()),
                     run_as_user_name: Some("winuser".to_string()),
                     ..Default::default()
-                })
+                }),
+                app_armor_profile: None
             }
         );
     }
diff --git a/crates/stackable-operator/src/builder/pod/volume.rs b/crates/stackable-operator/src/builder/pod/volume.rs
index 07eab3d..3347101 100644
--- a/crates/stackable-operator/src/builder/pod/volume.rs
+++ b/crates/stackable-operator/src/builder/pod/volume.rs
@@ -258,6 +258,7 @@ impl VolumeMountBuilder {
             read_only: self.read_only,
             sub_path: self.sub_path.clone(),
             sub_path_expr: self.sub_path_expr.clone(),
+            recursive_read_only: None,
         }
     }
 }
```

Reply:
We **technically** it's a breaking change as we re-export k8s-openapi, but so is the kube and k8s-openapi bump as well ;)
I'll test the bump locally

Reply:
`cargo test` at least passes with the above changes applied.